consumers/jira_c/gojira/jirautils.go

```diff
@@ -0,0 +1,162 @@
+// :param fieldType: the type of the field in Jira (single-value, multi-value, float)
+// :param values: list of values to be filled in
+// :return the appropriate interface for a CustomField, given the corresponding fieldType and value(s)
+func makeCustomField(fieldType string, values []string) interface{} {
+	switch fieldType {
+	case "single-value":
+		return map[string]string{"value": values[0]}
+	case "multi-value":
+		cf := []map[string]string{}
+		for _, v := range values {
+			cf = append(cf, map[string]string{"value": v})
+		}
+		return cf
+	case "float":
+		if f, err := strconv.ParseFloat(values[0], 64); err == nil {
+			return f
+		} else {
+			log.Fatalf("Error parsing float field-type: %v", err)
+		}
+	default:
+		return nil
+	}
+	return nil
+}
+
+// makeProjectField returns a jira.Project object with the given key value
+func makeProjectField(value string) jira.Project {
+	return jira.Project{
+		Key: value,
+	}
+}
+
+// makeIssueTypeField returns a jira.IssueType object with the given name value
+func makeIssueTypeField(value string) jira.IssueType {
+	return jira.IssueType{
+		Name: value,
+	}
+}
+
+// makeComponentsField returns a []jira.Components object with the given name value(s)
+func makeComponentsField(values []string) []*jira.Component {
+	components := []*jira.Component{}
+	for _, v := range values {
+		components = append(components, &jira.Component{Name: v})
+	}
+	return components
+}
+
+// makeAffectsVersionsField returns a []jira.AffectsVersion object with the given name value(s)
+func makeAffectsVersionsField(values []string) []*jira.AffectsVersion {
+	affectsVersions := []*jira.AffectsVersion{}
+	for _, v := range values {
+		affectsVersions = append(affectsVersions, &jira.AffectsVersion{Name: v})
+	}
+	return affectsVersions
+}
+
+// makeDescription creates the description of an issue's enhanced with extra information from the Dracon Result
+// depending on the tool used, diffrent fields are more suitable as description
+func makeDescription(draconResult map[string]string, extras []string) string {
+	desc := "This issue was automatically generated by the Dracon security pipeline.\n\n"
+	switch draconResult["tool_name"] {
+	case "gosec":
+		desc = desc + "*" + draconResult["title"] + "*" + "\n\n"
+	case "spotbugs":
+		desc = desc + "*" + draconResult["description"] + "*" + "\n\n"
+	case "bandit":
+		desc = desc + "*" + draconResult["description"] + "*" + "\n\n"
+	}
+
+	// Append the extra fields to the description
+	if len(extras) > 0 {
+		desc = desc + "{code:}" + "\n"
+		for _, s := range extras {
+			desc = desc + fmt.Sprintf("%s: %*s\n", s, 25-len(s)+len(draconResult[s]), draconResult[s])
+		}
+		desc = desc + "{code}" + "\n"
+	}
+	return desc
+}
+
+// makeSummary creates the Summary/Title of an issue
+// depending on the tool used, different fields are more suitable as summary
+func makeSummary(draconResult map[string]string) string {
+	switch draconResult["tool_name"] {
```

Comment:
same here: I think YAML configuration that you read in would be a good additional feature

Reply:
Will work on implementing this in the future